rclcpp/src/rclcpp/clock.cpp

```diff
@@ -130,13 +130,16 @@ Clock::get_triggered_callback_handlers(const TimeJump & jump)
 {
   std::vector<JumpHandler::SharedPtr> callbacks;
   std::lock_guard<std::mutex> guard(callback_list_mutex_);
-  for (auto wjcb = active_jump_handlers_.begin(); wjcb != active_jump_handlers_.end(); wjcb++) {
+  for (auto wjcb = active_jump_handlers_.begin(); wjcb != active_jump_handlers_.end(); ) {
```

Comment:
couldn't you do the loop with a `std::erase(std::remove_if()` call? 
http://en.cppreference.com/w/cpp/algorithm/remove

Then you don't have to deal with the indices yourself :)

Reply:
I switched it to a lambda with remove_if instead. It's actually a line or two longer, but should be more performant on the erase due to remove_if's reordering. Though I don't expect performance to be an issue here.